cli/src/main/java/org/jboss/gm/cli/Main.java

```diff
@@ -286,10 +287,12 @@ public Void call() throws ManipulationException {
             if (logger.isDebugEnabled()) {
                 logger.debug("Configuration reloaded with {}", configuration.dumpCurrentConfig());
             }
-
-            GroovyUtils.runCustomGroovyScript(logger, InvocationStage.FIRST, target, configuration, null, null);
         }
 
+        GroovyUtils.runCustomGroovyScript(logger, InvocationStage.FIRST, target, configuration, null, null);
```

Comment:
Was this a bug inside the `if` or does it need to have `groovyScripts` defined anyway?